32-fall-detector-and-panic-button/README.md

```diff
@@ -1,89 +1,86 @@
+
+## Swan Setup
 
-Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
+First connect your Blues Swan and Notecard to your Notecarrier-F.
 
-## Sparrow Setup
+1. Follow the steps in the [Notecard Quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#connect-your-notecard-and-notecarrier) to connect your Notecard LoRa to your Notecarrier-F.  Your antenna will only have one cable, and the Notecard LoRa only has one connection.
 
-### Quickstart
+2. Plug your Swan into the Feather headers on the Notecarrier F.
 
-Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) to get your Sparrow reference node paired with the gateway and associated with the Notehub project you just created. Note also that we'll only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the `hub.set` command in the quickstart.
+3. Attach the Swan to your computer with a Micro USB to USB-A cable, using the **Micro USB port on the Swan**.
 
-After you've completed the quickstart, leave the Notecarrier and essentials board powered and connected. These two devices will act as our gateway to Notehub, and we won't need to touch them again. The rest of this guide will focus on the reference node.
+![Notecard LoRa in Notecarrier F with antenna](images/notecarrier-notecard.jpg)
 
-### Hardware
+## Hardware Setup
 
 There are two primary pieces of hardware: the button and the accelerometer.
 
-The button will be wired to a GPIO pin on the reference node. This pin uses an internal pull-up so that the it reads a digital 1 when the button isn't pushed. When the button is pushed, it connects the corresponding GPIO pin to GND so that it reads a digital 0. Using the breadboard, button, and jumper wires, wire up the hardware as shown in the diagram below, connecting one side of the button to A2 and the other to the minus (-) rail of the breadboard. Make sure to then connect the minus rail to the reference node's GND port.
+The button will be wired to a GPIO pin on the Swan. This pin uses an internal pull-up so that the it reads a digital 1 when the button isn't pushed. When the button is pushed, it connects the corresponding GPIO pin to GND so that it reads a digital 0. Using the breadboard, button, and jumper wires, wire up the hardware as shown in the diagram below, connecting one side of the button to F_D12 and the other to the minus (-) rail of the breadboard. Make sure to then connect the minus rail to one of the GND pins on the Notecarrier-F.
 
-![breadboard button wiring](assets/breadboard_button.jpg "Breadboard Button Wiring")
+![breadboard button wiring](images/breadboard_button.jpg "Breadboard Button Wiring")
```

Comment:
The image still has `A2` in it and not `F_D12`.

Reply:
Thanks - I will generate a new image.